wrappers/swift/main.swift

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+let props = EventProperties(name:"TestEvent")
+props.setProperty("PropName", withValue: ["Type":"SwiftWrappers"])
+props.setProperty("PropWithPII", withInt64Value: Int64(30), withPiiKind: ODWPiiKind.distinguishedName)
```

Comment:
Are consumers going to use Obj-C constants/enums, or are you going to setup new ones for Swift?

Reply:
I am intending to let consumers use the obj-c enums instead of creating new ones in swift